content/api/commands/intercept.md

```diff
@@ -170,116 +94,153 @@ The `routeHandler` defines what will happen with a request if the [routeMatcher]
-// 'q=expected+terms' responds, this 'cy.wait' will resolve
-cy.wait('@search')
+cy.intercept('GET', '**/users').as('getAllUsers')
+cy.intercept('POST', '**/users').as('createUser')
 ```
 
-#### Using the yielded object
+<!-- TODO move this -->
+`cy.intercept` can be used to passively listen for matching routes and apply aliases to them without manipulating the request or its response in any way. As you'll see, this alone is powerful as it allows you to wait for these requests, resulting in more reliable tests.
+
+### Waiting on a request
 
-Using [cy.wait()](/api/commands/wait) on a `cy.intercept()` route alias yields an interception object which represents the request/response cycle:
+We can build upon the previous examples of defining an aliased route with [cy.wait()](/api/commands/wait) in order to wait for the request to be sent and a response to be received before executing the next command.
 
 ```js
-cy.wait('@someRoute').then((interception) => {
-  // 'interception' is an object with properties 'id', 'request' and 'response'
-})
+cy.intercept('GET', '**/users').as('getAllUsers')
+cy.intercept('POST', '**/users').as('createUser')
+
+// remember the `@` prefix when referencing aliases
+cy.get('button.createUser').click()
+cy.wait('@createUser')
+// the commands below will not execute until a response is received from `POST /users`
+cy.get('#field.firstName').should('equal', 'Adam')
+
 ```
 
-You can chain [`.its()`](/api/commands/its) and [`.should()`](/api/commands/should) to assert against request/response cycles:
+#### Making assertions on a request/response
+
+When invoked with a route alias, [`cy.wait`](/api/commands/wait) returns an object that includes `request` and `response` which we can make assertions on:
 
 ```js
-// assert that a request to this route was made with a body that included 'user'
-cy.wait('@someRoute').its('request.body').should('include', 'user')
+// assert that the request body includes 'user'
+cy.wait('@createUser').its('request.body').should('include', 'user')
 
-// assert that a request to this route received a response with HTTP status 500
-cy.wait('@someRoute').its('response.statusCode').should('eq', 500)
+// assert that the response HTTP status is 201 (Created)
+cy.wait('@createUser').its('response.statusCode').should('eq', 201)
+```
+
+<!-- make this example better
+what's actually in the ic object? -->
+### Using the yielded object
+You could also access the entire object which represents the request/response cycle that was intercepted:
 
-// assert that a request to this route received a response body that includes 'id'
-cy.wait('@someRoute').its('response.body').should('include', 'id')
+```js
+cy.wait('@createUser').then((ic) => {
+  const { request, response, id, ...rest } = ic
+  // ...
+})
 ```
 
-#### Aliasing individual requests
 
+#### Aliasing individual requests
+<!-- does this have other use cases outside of graphQL? -->
```

Comment:
I might create a graphQL section further down for all this. The first example in this section uses `routeHandler` which we haven't introduced yet.